### PR TITLE
chrome_url_overrides update for Opera

### PR DESCRIPTION
**Pull request by eiselzby**
Browser compatibility update for chrome_url_overrides in Opera

### CODE REVIEW

File: webextensions/manifest/chrome_url_overrides.json [hunk 1]
```diff
@@ -46,10 +46,7 @@
                 ]
               },
               "opera": {
-                "version_added": true,
-                "notes": [
-                  "If two or more extensions both define a custom new tab page, then in Firefox the first extension to run wins. In Chrome and Opera, the last extension wins."
-                ]
+                "version_added": false
```
Comment (wbamberg) [inline webextensions/manifest/chrome_url_overrides.json]: Please also change "In Chrome and Opera" in the Firefox notes to "In Chrome".

**Comment by wbamberg** [comment]
Requesting review from @Elchi3 for the gitignore changes.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 /node_modules/
+/.idea/
+package-lock.json
```
Comment (Elchi3) [inline .gitignore]: This suggest that package-lock.json files should be included in version control. We don't have a package-lock.json, but we might want to add it.
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

File: .gitignore [hunk 3]
```diff
@@ -1 +1,3 @@
 /node_modules/
+/.idea/
```
Comment (Elchi3) [inline .gitignore]: What is this for? Intellij?

Comment (eiselzby) [reply .gitignore]: It's from Jetbrain's Webstorm.

**Comment by eiselzby** [comment]
What are correct browser versions for Microsoft Edge? Microsoft uses versions like MS Edge 40.15063.674.0 / Microsoft Edge (40) / Windows 10 (15063), but none of them are accepted by CI tests.

File: .gitignore (hunk 3 above)
Comment (Elchi3) [reply .gitignore]: Can you add this to your own global gitignore maybe?
We probably don't want add this for every IDE.

**Comment by Elchi3** [comment]
Edge release versions are: 12, 13, 14, 15, and 16.
We store a list of browser releases here: https://github.com/mdn/browser-compat-data/blob/master/browsers/browsers.json#L256

File: webextensions/manifest/chrome_url_overrides.json [hunk 4]
```diff
@@ -27,29 +27,23 @@
               "chrome": {
                 "version_added": true,
                 "notes": [
-                  "If two or more extensions both define a custom new tab page, then in Firefox the first extension to run wins. In Chrome and Opera, the last extension wins."
+                  "If two or more extensions both define a custom new tab page, then in Firefox the first extension to run wins. In Chrome wins the last extension."
```
Comment (wbamberg) [inline webextensions/manifest/chrome_url_overrides.json]: Should be "In Chrome the last extension wins."

File: webextensions/manifest/chrome_url_overrides.json [hunk 5]
```diff
@@ -27,29 +27,23 @@
...
               },
               "edge": {
-                "version_added": false
+                "version_added": "15"
               },
               "firefox": {
                 "version_added": "54",
                 "notes": [
-                  "If two or more extensions both define a custom new tab page, then in Firefox the first extension to run wins. In Chrome and Opera, the last extension wins."
+                  "If two or more extensions both define a custom new tab page, then in Firefox the first extension to run wins. In Chrome wins the last extension."
```
Comment (wbamberg) [inline webextensions/manifest/chrome_url_overrides.json]: Should be "In Chrome the last extension wins."

**Pull request merged**